src/App.vue

```diff
@@ -614,14 +614,16 @@ export default class App extends Mixins(
           }
         )
       }
+
+      console.log('*** got data')
```

Comment:
The issue that's bugging me atm is that the dissolution Complete Resolution and Complete Affidavit steps don't show a valid checkmark when you resume a draft, even though those pages are valid. Example:

https://business-create-dev.apps.silver.devops.gov.bc.ca/businesses/create/dissolution-define-dissolution?id=BC0870918&accountid=2288

Unfortunately this doesn't happen if you reload the page, or locally. I believe there's a timing issuel possibly the pages render (and decide what's valid) before the data is loaded.

So this is an attempt to verify that we get the data before the pages render. This is temporary code that I remove later.

Reply:
I can no longer reproduce this issue in Dev. There are several possibilities:

1. setting haveData in the try block instead of the finally block solved it
2. calling the step validity method from mounted instead of created, and after letting things settle a bit, solved it
3. the timing issue still exists but isn't see atm due to code changes
4. the stars aligned differently right now and the issue still exists but is sporadic.

🤷‍♂️

Reply:
Ah nuts, I just reproduced it in Dev, so it's possibility 3 or 4.

HOWEVER, my logging shows this...

![image](https://user-images.githubusercontent.com/10002889/156654605-fa18b9bc-4741-433f-be82-481ea837268a.png)

... and I know that parseDissolutionDraft() clears (initializes) validationItemDetails[], so that's why you can see the step checkmark for a split second and then disappears (ie, it thinks the page is not valid).

Since fetchData() (which displays "got data") is not being called from an error dialog, it means the route has changed twice. Hmm... 🤔